contentctl/objects/abstract_security_content_objects/detection_abstract.py

```diff
@@ -223,42 +240,40 @@ def cve_enrichment_func(self, info:ValidationInfo):
                 raise ValueError(f"{e}")
         self.cve_enrichment = enriched_cves
         return self
-    
 
+    # TODO (cmcginley): can we tighten this typing? is this field even being used?
     splunk_app_enrichment: Optional[List[dict]] = None
```

Comment:
Is this field even used?

Reply:
This is legacy code which is related to DocGen Functionality, which is no longer exposed. I have raised an internal issue with the team to see if DocGen can/should be removed. If so, that will be a much larger PR but should be a nice cleanup to numerous files :) 
As such, I am going to leave this comment unresolved, but no relevant changes should be part of THIS PR.